src/main/java/com/gentics/elasticsearch/client/okhttp/ElasticsearchOkClient.java

```diff
@@ -35,13 +36,28 @@
 	 *            Server port
 	 */
 	public ElasticsearchOkClient(String scheme, String hostname, int port) {
-		super(scheme, hostname, port);
-		this.client = createClient();
+		this(scheme, hostname, port, 10_000);
```

Comment:
Use default ok http client timeouts instead

Reply:
That is actually the default value. There's no Constant for the timing sadly, only hardcoded.
See: https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/OkHttpClient.java#L267
https://github.com/square/okhttp/blob/master/okhttp/src/main/java/okhttp3/OkHttpClient.java#L474